AdminWebsite/AdminWebsite/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -2697,6 +2745,10 @@ export class BookNewHearingRequest implements IBookNewHearingRequest {
                 this.endpoints = [] as any;
                 for (let item of _data['endpoints']) this.endpoints!.push(EndpointRequest.fromJS(item));
             }
+            if (Array.isArray(_data['linked_participants'])) {
```

Comment:
This isn't needed - the items in _data are already linked participants, they don't need transforming in TS, unlike in C#

Reply:
this file (api-client.ts) is auto generated